app/navigation/index.js

```diff
@@ -20,9 +15,7 @@ const store = createStore()
 class App extends Component {
   render () {
     return (
-      <Provider store={store}>
```

Comment:
Jeżeli będziesz korzystać z reduxa to jest Ci potrzebny ten `Provider` i logika do tworzenia store'a.